rimage/depth_map.go

```diff
@@ -117,41 +116,17 @@ func (dm *DepthMap) SubImage(r image.Rectangle) *DepthMap {
 // or if it can be converted into one.
 func ConvertImageToDepthMap(img image.Image) (*DepthMap, error) {
 	switch ii := img.(type) {
-	case *LazyEncodedImage:
```

Comment:
Why don't we need to handle the lazy case here?

Reply:
we do, when I undid my changes I forgot this part, adding it back

Reply:
adding back the test cases I forgot, so that will make sure I don't forget about this one again